cmake/LICENSE

```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2016, Los Alamos National Security, LLC
```

Comment:
See https://github.com/quinoacomputing/quinoa/pull/151, https://github.com/quinoacomputing/cmake-modules/pull/11, and https://github.com/quinoacomputing/quinoa-tpl/pull/17.